src/peer/meshRoom.js

```diff
@@ -57,6 +57,22 @@ class MeshRoom extends Room {
     this.emit(MeshRoom.MESSAGE_EVENTS.getPeers.key, data);
   }
 
+  /**
+   * Gets the RTCPeerConnection objects used by Connections in this room. Modifying the
+   * RTCPeerConnection can cause unexpected errors. Suggested for advanced users only.
+   * @return {Object} - An object containing the media RTCPeerConnections used by this room with the peerId as the key.
+   */
+  getRTCPeerConnections() {
+    // TODO: add a `type` argument specifying connection type after we implement data connections in meshRoom.
+    const peerConnections = {};
+    for (const [peerId, connections] of Object.entries(this.connections)) {
+      if (Array.isArray(connections) && connections.length > 0) {
```

Comment:
Type of `connections` is always Array, isn't it?

Reply:
Yeah I was trying to protect against some weird case where it's undefined or something.
I'll remove it since I guess it's unnecessary.

Reply:
I think it's ok to remove it.
(and also I know there're so many points we should protect that...unfortunately..)